client/src/components/EditProfile/AvailabilityTab/AvailabilityTab.tsx

```diff
@@ -139,7 +139,7 @@ const AvailabilityTab = (): JSX.Element => {
 
   useEffect(() => {
     if (updating) {
-      editAvailability(mockId, availability).then((data) => console.log(data));
+      editAvailability(mockId, availability);
```

Comment:
Why don't we update the snack bar with a relevant message on successful/unsuccessful update?